app/src/main/java/seemoo/fitbit/activities/WorkActivity.java

```diff
@@ -144,6 +144,9 @@ public boolean onNavigationItemSelected(MenuItem menuItem) {
                             case R.id.nav_devices:
                                 buttonDevices();
                                 break;
+                            case R.id.nav_reconnect:
+                                buttonReconnect();

```

Comment:
could we not just call connect() directly here?

Reply:
We could, yes, but I created the buttonReconnect()-Method for the sake of uniformity, because all other buttons have a corresponding button-Method.

Reply:
I would prefer precise code here instead of uniformity. All other button methods have multiple commands. More code = more complexity